justfile

```diff
@@ -12,10 +12,9 @@ test +projects=PROJECTS:
         poetry -C ${p}/ run nox -f ${p}/noxfile.py -s coverage
     done
 
-
 # Create a release
 release project version:
-    @echo "This currently is a stub, in the future the workflow probably will look somthing like this:"
+    @echo "This currently is a stub, in the future the workflow probably will look something like this:"
```

Comment:
I think the `echo`s can be removed they have been there as dummy until replaced with an actual implementation.

Reply:
I was not sure whether we should prefer bare calls to poetry or nox. 
As there is also nox session prepare-release.

But see next push for removed echo commands.